### ISSUE TITLE
Cannot upgrade past 6.8.19

### ISSUE THREAD

**Issue by gianlucabonetti** [Intermesh/groupoffice#1078]
Hello

There is an error while trying to upgrade past 6.8.19 happening with 6.8.20-21-22

```
Error: Exception in /var/www/clients/client8/web1/web/groupoffice-6.8.22/go/core/Installer.php at line 787: HY000: SQLSTATE[HY000]: General error: 1709 Index column size too large. The maximum column size is 767 bytes
Query: ALTER TABLE `addressbook_contact` CHANGE `lastName` `lastName` VARCHAR(255) DEFAULT NULL;
```

![image](https://github.com/Intermesh/groupoffice/assets/38787379/2e49fb6b-ad7d-454c-af0f-40134e403e7b)

**Comment by mschering** [comment]
Hi,

I think it's because you're running a slightly older version of mysql. Next release will have a smaller field again so it will work on your db too.

Best regards,
Merijn

**Comment by gianlucabonetti** [comment]
Hello
It is running on Debian 10 (LTS) on MariaDB 10.5
Thanks
Gianluca

**Comment by mschering** [comment]
Ah ok, then I was wrong. It might be the row format that is different then. Anyway, next release will fix it.

**Comment by gianlucabonetti** [comment]
No worries, I was just providing more information, as my ticket didn't have this (and quite important detail)
Thanks for your support
Gianluca

**Comment by gianlucabonetti** [comment]
I confirm this is solved with released 6.8.23.
Thanks for your support
